scripts/bench.sh

```diff
@@ -5,9 +5,14 @@ MYDIR=$(realpath "$(dirname "$0")")
 cd "$MYDIR"
 cd ..
 
-NIGHTLY_TOOLCHAIN=${NIGHTLY_TOOLCHAIN:-nightly}
+NIGHTLY_TOOLCHAIN=${NIGHTLY_TOOLCHAIN:-nightly-2024-01-01}
```

Comment:
Maybe use a more recent nightly like `2024-09-01`?

Reply:
Alas, recent nightly version do not work. That is another problem why we're pinned. Also I've checked that on stable not just binary size is smaller, but benchmarks perform %2-3 worse